core/src/gql/game/game.resolver.ts

```diff
@@ -3,14 +3,21 @@ import { SkillGroupObject } from '../skill_group/skill_group.object';
 import { GameObject } from './game.object';
 import { GameRepository } from '../../db/game/game.repository';
 import { UseGuards } from '@nestjs/common';
-import { AuthorizeGuard } from '../../auth/authorize/authorize.guard';
+import { GameModeObject } from '../game_mode/game_mode.object';
+import { AuthPossession, AuthZGuard, UsePermissions } from 'nest-authz';
+import { Resource, ResourceAction } from 'src/db/authorize/constants';
 
 @Resolver(() => GameObject)
 export class GameResolver {
   constructor(private readonly gameRepo: GameRepository) {}
 
   @Query(() => [GameObject])
-  @UseGuards(AuthorizeGuard())
+  @UseGuards(AuthZGuard)
+  @UsePermissions({
+    resource: Resource.Game,
+    action: ResourceAction.Read,
+    possession: AuthPossession.ANY,
```

Comment:
More of a question, but can we extend this possession concept to properly account for franchise / team / league scopings?

Reply:
I'm fairly certain this possession concept will work for that purpose as is, but we can always add another field to the policy if we find we need another dimension.